galera/src/replicator_str.cpp

```diff
@@ -342,6 +342,38 @@ ReplicatorSMM::donate_sst(void* const         recv_ctx,
     /* The fix to codership/galera#284 may break backward comatibility due to
      * different (now correct) interpretation of retrun value. Default to old
      * interpretation which is forward compatible with the new one. */
+    /* The fix to codership/galera#284 may break backward comatibility due to
+     * different (now correct) interpretation of retrun value. Default to old
+     * interpretation which is forward compatible with the new one. */
+
+    /* Backward compatibility means: WSREP API v25 + API client
+     * (sst_donate_cb) that returns positive values in case of success. 
+     * Before commit 3fd754250bc1ef4e0459848d4a390609f311c45a (galera),
+     * results >= 0 returned by API client were considered to be success.
+     * Before commit 6d1e77ad1399e9558462f56fa9e36a3217622bb1 (wsrep client), 
```

Comment:
Commit `6d1e77ad13` doesn't exist in wsrep-API repo. Please replace `wsrep client` with `pxc` 
```suggestion
     * Before commit 6d1e77ad1399e9558462f56fa9e36a3217622bb1 (pxc), 
```

Also remove all the trailing white spaces in the newly added comment block.

Reply:
writing 'wsrep client' I have in mind "the one who uses Galera library. It is not necessarely PXC. From Galera point of view we know nothing about PXC. In fact the change was in Codership's mysql, not PXC. I changed it to 'api client'

Reply:
In that sense, I also agree with you.